CHANGELOG.rst

```diff
@@ -141,11 +167,6 @@ Individual components
 - `renku-ui 3.8.1 <https://github.com/SwissDataScienceCenter/renku-ui/releases/tag/3.8.1>`_
 
 
-0.29.0
-------
-
-Renku ``0.29.0`` introduces new UI features and a PostgreSQL DB for triples-generator.
```

Comment:
Was this left in accidentally?

Reply:
I think it was removed but it shouldn't. As a result, the `0.29.0` headline was missing. I brought it back.